dds/idl/rapidjson_generator.cpp

```diff
@@ -242,31 +287,33 @@ namespace {
         }
         strm <<
           ip << "}\n";
-      } else if (pt == AST_PredefinedType::PT_char) {
+      } else switch (pt) {
```

Comment:
```suggestion
      } else {
        switch (pt) {
```

Reply:
I was trying to avoid raising the indent, but I guess that would be better style-wise.